### DISCUSSION TITLE
Unable to authenticate in Rasbpian

### DISCUSSION THREAD

**Discussion by zeverson** [homebridge-simplisafe3/homebridge-simplisafe3#420] [Q&A]
Trying to install it on Raspbian, using Chromium, Firefox, and Vivaldi. After entering the 2FA, a popup window appears:

Open xdg-open?
https://auth.simpl/[isafe.com](http://isafe.com/) wants to open this application

If I select Open xdg-open, a new browser window opens displaying the browser's home page. There's no error in view console.

If I select Cancel, nothing happes.

Any idea how I can get Homebridge's SimpliSafe working? Thank you!

**Comment by shamoon** [comment]
Are you using cli?
You can just copy and paste the URL it gives you into a browser to start the process

**Reply by zeverson** [reply]
No, I can't install the plugin that way. Entering the command gets me an error starting with "npm does not support Node.js v10.24.0.

**Reply by shamoon** [reply]
Thats not an issue with this plugin, and seems pretty clear no?

In general the webUI is usually easier for most folks

**Reply by zeverson** [reply]
Yeah, web UI was what I was trying in the first place, which got me that xdg message. I only tried cli when the webUI didn't work.

**Reply by shamoon** [reply]
Oh I see. Yea you dont want to open xdg. When you run the login process the SS website will try to redirect you to a URL, in most browsers that will just fail (its supposed to open the iOS app) but seems like on your machine for whatever reason its trying to open that url with open-xdg.

But anyway, you want to copy that URL (usually its shown in the browser error console) and paste it back into homebridge.

See 1.iii https://github.com/homebridge-simplisafe3/homebridge-simplisafe3?tab=readme-ov-file#simplisafe-authentication
